plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/EJBMediatorExtFactory.java

```diff
@@ -50,6 +50,73 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
             mediator.setClassName(attributeValue.trim());
         }
 
+        attributeValue = elem.getAttributeValue(new QName(EJBConstants.STATEFUL));
+        if (Boolean.valueOf(attributeValue)) {
+            attributeValue = elem.getAttributeValue(new QName(EJBConstants.BEAN_ID));
+            if (attributeValue != null) {
+                mediator.setBeanId(new ValueExtFactory().createValue(EJBConstants.BEAN_ID, elem));
+            }
+        }
+
+        boolean remove;
+        attributeValue = elem.getAttributeValue(new QName(EJBConstants.REMOVE));
+        remove = Boolean.valueOf(attributeValue);
+        if (remove) {
+            mediator.setRemove(true);
+        }
+
+        String targetValue = elem.getAttributeValue(new QName(BeanConstants.TARGET));
+        if (targetValue != null) {
+            mediator.setTargetValue(targetValue);
+        }
+
+        attributeValue = elem.getAttributeValue(new QName(EJBConstants.JNDI_NAME));
+        if (attributeValue != null) {
+            mediator.setJndiName(attributeValue);
+        }
+
+        OMElement argumentsElem = elem
+                .getFirstChildWithName(new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, EJBConstants.ARGS));
+
+        if (argumentsElem != null) {
+
+            @SuppressWarnings("rawtypes")
```

Comment:
Why are we supressing warnings? Besides, @SuppressWarnings("unchecked") is preferred acorss IDEs

Reply:
Fixed ...